servers_v7.json

```
@@ -77,7 +77,7 @@
   },
   {
     "name": "The Devil",
-    "address": ["xem8k5.mindustry.top:8169", "test.xem8k5.top:8920", "other.xem8k5.top:10936"]
+    address":["xem8k5.mindustry.top:8169","test.xem8k5.top:8920","other.xem8k5.top:10795","other.xem8k5.top:10936"]
```

```suggestion
    "address": ["xem8k5.mindustry.top:8169", "test.xem8k5.top:8920", "other.xem8k5.top:10795", "other.xem8k5.top:10936"]
```
You forgot an apostrophe and the style isn't consistent with the rest of the file.